### PR TITLE
[WIP] Added missing link for secret binding

### PR DESCRIPTION
**Pull request by amarie401**
Fixed issue where binding confirmation was not a link. This would close #1908

https://github.com/openshift/origin-web-console/issues/1908

Note: The issue stated that it should be a link to that secret but the href currently used is ```{{ctrl.secretHref}}``` which is just ```'<'``` in  ```origin-web-common.js```. Not sure if that is correct or if it should be a different path. 

@jeff-phillips-18  @spadgett

### CODE REVIEW

File: src/components/binding/bindResults.html [hunk 1]
```diff
@@ -20,7 +20,8 @@ <h3 class="bind-message">
       The binding operation created the secret
       <a ng-if="ctrl.secretHref && 'secrets' | canI : 'list'"
          ng-href="{{ctrl.secretHref}}">{{ctrl.binding.spec.secretName}}</a>
-      <span ng-if="!ctrl.secretHref || !('secrets' | canI : 'list')">{{ctrl.binding.spec.secretName}}</span>
+      <a ng-if="!ctrl.secretHref || !('secrets' | canI : 'list')"
+         ng href="{{ctrl.secretHref}}">{{ctrl.binding.spec.secretName}}</a>
```
Comment (dtaylor113) [inline src/components/binding/bindResults.html]: Is this supposed to be ```ng-href```?

Comment (amarie401) [inline src/components/binding/bindResults.html]: @dtaylor113 whoops! yes!

Comment (amarie401) [inline src/components/binding/bindResults.html]: @dtaylor113 will fix :)

**Comment by amarie401** [comment]
@dtaylor113 fixed it :)

**Comment by jeff-phillips-18** [comment]
@amarie401 What you have `"{{ctrl.secretHref}}"` is correct.

File: dist/origin-web-common-ui.js [hunk 2]
```diff
@@ -113,7 +113,8 @@ hawtioPluginLoader.addModule('openshiftCommonUI');
     "      The binding operation created the secret\n" +
     "      <a ng-if=\"ctrl.secretHref && 'secrets' | canI : 'list'\"\n" +
     "         ng-href=\"{{ctrl.secretHref}}\">{{ctrl.binding.spec.secretName}}</a>\n" +
-    "      <span ng-if=\"!ctrl.secretHref || !('secrets' | canI : 'list')\">{{ctrl.binding.spec.secretName}}</span>\n" +
+    "      <a ng-if=\"!ctrl.secretHref || !('secrets' | canI : 'list')\"\n" +
+    "         ng-href=\"{{ctrl.secretHref}}\">{{ctrl.binding.spec.secretName}}</a>\n" +
```
Comment (jeff-phillips-18) [inline dist/origin-web-common-ui.js]: The ng-if command is correct and this should remain a span not an anchor.

**Comment by amarie401** [comment]
@dtaylor113  @jeff-phillips-18  I will close PR this as the problem was in origin-web-console and not origin-web-common.

**Pull request closed without merging**